docs/en/stack/ml/get-started/ml-gs-jobs.asciidoc

```diff
@@ -3,6 +3,11 @@
 [[sample-data-jobs]]
 = Create sample {anomaly-jobs} in {kib}
 
+IMPORTANT: The results you can see in the screenshots of this page might be 
+different than the actual values you get when using the sample data sets. This 
+behavior is expected as the data points in the data sets might change in 
```

Comment:
In case changes happen within the same version (e.g. between 8.6.0 and 8.6.1), something like this might be simpler:
```suggestion
behavior is expected as the data points in the data sets might change over time.
```